src/middlewared/middlewared/plugins/failover_/event.py

```diff
@@ -644,6 +644,7 @@ def vrrp_master(self, job, fobj, ifname, event):
                     'However, other zpools imported so the failover process continued.'
                 )
         else:
+            self.middleware.create_task(self.middleware.call('zfs.pool.ddt_prefetch_pools'))
```

Comment:
I don't think we should do this. I have no idea how long this takes or the performance implications on the zpools but not to fond of this change. We need @amotin and or @ixhamza, @usaleem-ix 's input.

Reply:
We could run it asynchronously without waiting for completion.  May be even few seconds after pool import, so that other more important processes had time to start.  The prefetch may take some time, but without it on a pool with large DDT/BRT performance may suffer for hours until they get fully loaded on-demand.

Reply:
This is being run in the background and we are not waiting for it to complete execution or halt the failover event.

Reply:
Yeah I know it's being run in background but I was talking about whether this task kicks off IO on the zpool that causes performance degradation.

Reply:
Ah no, i meant to clarify to @amotin that it is running asynchronously and not blocking at this point. Based off that, please advise @amotin if you think we should let it be and keep doing it here?